src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/corporate/CorporateMappingDtos.kt

```diff
@@ -37,72 +37,62 @@ data class CorporateMappingIdDto(
 )
 
 class CorporateMappingDto(
```

Comment:
All DTOs are currently identical; might decide to remove all of them - but for now push everything in base class so we have simplify to toMapping function